.gitignore

```diff
@@ -6,4 +6,11 @@ models--*
 *.log
 *.info
 *.json
-*.sh
\ No newline at end of file
+*.sh
```

Comment:
why ignore .sh files if you are committing 2 in this PR?

Reply:
Changed to ignore the auto-generate .e and .o files after runs